linera-service/tests/end_to_end_tests.rs

```diff
@@ -978,18 +980,20 @@ async fn test_wasm_end_to_end_matching_engine(config: impl LineraNetConfig) {
 #[cfg_attr(feature = "scylladb", test_case(LocalNetTestingConfig::new(Database::ScyllaDb, Network::Grpc) ; "scylladb_grpc"))]
 #[cfg_attr(feature = "aws", test_case(LocalNetTestingConfig::new(Database::DynamoDb, Network::Grpc) ; "aws_grpc"))]
 #[cfg_attr(feature = "kubernetes", test_case(SharedLocalKubernetesNetTestingConfig::new(Network::Grpc, None) ; "kubernetes_grpc"))]
+#[cfg_attr(feature = "gcp_net", test_case(GCPNetTestingConfig::new() ; "gcp_net_grpc"))]
 #[test_log::test(tokio::test)]
 async fn test_wasm_end_to_end_amm(config: impl LineraNetConfig) {
     use amm::{AmmAbi, Parameters};
     use fungible::{FungibleTokenAbi, InitialState};
     let _guard = INTEGRATION_TEST_GUARD.lock().await;
 
+    let maybe_faucet = config.maybe_faucet();
     let (mut net, client_admin) = config.instantiate().await.unwrap();
```

Comment:
Let's remove `config.maybe_faucet` and just agree that `config.instantiate` returns *a* client (and not necessarily the one controlling the admin chain)